### ISSUE TITLE
Allow for variants filtering using any panel in database

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#2006]
Do not restrict variant filtering to the case-specific gene panels.

**Comment by northwestwitch** [comment]
I wonder if we should include also old versions of the same pane..

**Comment by dnil** [comment]
Clearly! There may be a case for arbitrary lists based on e.g. old HPO searches. A suggestion might be to let this merge into the gene list upload mechanism, so that both uploaded lists and panels can be selected?

**Comment by dnil** [comment]
Any thoughts on good ways to indicate what genes where on the given list, but were not analysed by MIP? Or just as-is with a warning?

**Comment by northwestwitch** [comment]
> A suggestion might be to let this merge into the gene list upload mechanism, so that both uploaded lists and panels can be selected?

I think we don't have really a choice here: these other panels must be translated into genes before we can search using them. I'll try to propose something!

**Comment by northwestwitch** [comment]
> Any thoughts on good ways to indicate what genes where on the given list, but were not analysed by MIP/BALSAMIC? Or just as-is with a warning?

It escalates quickly I see .. 😆

**Comment by dnil** [comment]
Self-assigning is dangerous business.. 😆
